src/azure/azureController.ts

```diff
@@ -279,4 +315,20 @@ export class AzureController {
 		}
 	}
 
+
+	/**
+	 * Verifies if the token still valid, refreshes the token for given account
+	 * @param session
+	 */
+	public async checkAndRefreshToken(
+		session: mssql.IAzureAccountSession,
+		accountStore: AccountStore): Promise<void> {
+		const currentTime = new Date().getTime() / 1000;
```

Comment:
what does / 1000 mean?

Reply:
copy the code somewhere else in mssql extension which we refresh token when it's expired. getTime is in milliseconds and to compare to expire it needs to get converted